applications/ShapeOptimizationApplication/tests/mapper_test/run_test.py

```diff
@@ -1,8 +1,9 @@
 # Import Kratos core and apps
-from KratosMultiphysics import *
-from KratosMultiphysics.ShapeOptimizationApplication import *
+import KratosMultiphysics as km
```

Comment:
usually we use this:

```suggestion
import KratosMultiphysics as KM
```